db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -953,7 +953,7 @@ class NonBatchedOpsStressTest : public StressTest {
 
       thread->stats.AddGets(1, 1);
 
-      if (!FLAGS_skip_verifydb) {
+      if (!FLAGS_skip_verifydb && !read_older_ts) {
```

Comment:
I think I didn't add this since we don't support UDT in `PutEntity()`, yet, but I just realized that we still need this as we are verifying the plain value as wide column values.